configure.ac

```
@@ -1412,6 +1412,17 @@ if test "$cross_compiling" = yes; then
 	RUNSHARED=
 fi
 
+AC_ARG_VAR(HOSTRUNNER, Program to run CPython for the host platform)
+AC_MSG_CHECKING(HOSTRUNNER)
+if test -z "$HOSTRUNNER"
+then
+	AS_CASE([$ac_sys_system/$ac_sys_emscripten_target],
+  [Emscripten/node*], [HOSTRUNNER='node'],
+  [HOSTRUNNER='']
+)
+fi
+AC_MSG_RESULT($HOSTRUNNER)
```

How does the code work for you without AC_SUBST?
```suggestion
AC_SUBST([$HOSTRUNNER])
AC_MSG_RESULT([$HOSTRUNNER])
```

---

I think I was using `make ... HOSTRUNNER=...` 😅